src/frontend/js/app/AppNavbar/UserNavbarItem.js

```diff
@@ -1,23 +1,34 @@
-import React, { PropTypes } from 'react';
+import React, { Component, PropTypes } from 'react';
 
-const UserNavbarItem = ({ user, signOut }) => (
-    <ul className="nav navbar-nav pull-xs-right">
-        <li className="nav-item dropdown">
-            <a
-                aria-expanded="false"
-                aria-haspopup="true"
-                className="nav-link dropdown-toggle"
-                data-toggle="dropdown"
-                role="button"
-            >
-                {user.email}
-            </a>
-            <div className="dropdown-menu">
-                <a className="dropdown-item" href="/sign-out" onClick={signOut} role="button">Sign out</a>
-            </div>
-        </li>
-    </ul>
-);
+class UserNavbarItem extends Component {
+    signOut = (event) => {
+        event.preventDefault();
```

Comment:
What does the preventDefault prevent? The href? If so why put a link with an href in the first place?